core/prelude-editor.el

```diff
@@ -285,8 +285,8 @@ The body of the advice is in BODY."
 (require 'dired-x)
 
 ;; ediff - don't start another frame
-(require 'ediff)
-(setq ediff-window-setup-function 'ediff-setup-windows-plain)
+;; (require 'ediff)
```

Comment:
The `setq-default` didn't require a variable be declared, so remove unnecessary required.